c++/src/kj/compat/tls.c++

```diff
@@ -369,6 +369,18 @@ private:
     switch (cmd) {
       case BIO_CTRL_FLUSH:
         return 1;
+      case BIO_CTRL_INFO:
+      case BIO_CTRL_EOF:
+        // Queries for info and EOF
+        return 0;
+#if OPENSSL_VERSION_NUMBER >= 0x30000000L && \
```

Comment:
Maybe this could just be `#ifdef BIO_CTRL_GET_KTLS_SEND`? That way if/when BoringSSL adds support, we won't need to update this.

Reply:
Good idea, sure.